english/forest/drag/drag.md

```diff
@@ -0,0 +1,3 @@
+You drag yourself further into the woods. Along the way, you disturb a nest of rattle snakes. Before you can get away, they lunge at you and inject you with their venom. You die in even more pain than you were before.
+
+&mdash; By [@svdb](https://twitter.com/svdb)
```

Comment:
Could you remove these attributions? The mdash is causing a bug in Casey when she tries to remove the pull request, and I'd rather not have every author put their twitter handle into the story. If people want to see who wrote this section, they can look at the commit history and find your GitHub profile.